server/models/chef.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const chefSchema = new Schema({
+  profilePicURL: String,
+  chefBio: {
+    tagLine: { type: String, required: true },
+    fullBio: { type: String, required: true },
+  },
```

Comment:
I'm wondering about your thoughts on this, but I'm thinking once someone has signed up to be a chef, they no longer need their regular profile and it can be overwritten with their chef profile. In that case we could just have tagLine and Bio in the user model and remove it from here.

Reply:
Yes, I debated that myself and am fine with your suggestion.  If we do that I am not sure we need the tag line.  We can just use the first line of the bio for the chef's tag line.